### PR TITLE
some progress

### PR DESCRIPTION
**Pull request by tester101010**
i'm some develop

### CODE REVIEW

File: ConsoleApp_06_avgust_2021_practic/Program.cs [hunk 1]
```diff
@@ -23,13 +23,11 @@ static void Main(string[] args)
 
             //Console.WriteLine(ClassPerson.ClassFIO.FullName());
             #endregion
-            //Console.WriteLine(ClassPerson.);
+            //Console.WriteLine(ClassPerson.ClassFIO.FullName());
```
Comment (Bloodhount) [inline ConsoleApp_06_avgust_2021_practic/Program.cs:26]: this is good variant

File: ConsoleApp_06_avgust_2021_practic/Program.cs [hunk 2]
```diff
@@ -23,13 +23,11 @@ static void Main(string[] args)
...
 
-            //computation.Calculator.Add(x3,x2);
             MethodCalculateNum1OrNum2(x1, x3);
 
-            //Console.WriteLine(computation.StringCalc.
-            //    StringLength("jnjkldghdfghdfghdfhdfnjkh"));
+            //Console.WriteLine(computation.StringCalc.StringLength("jnjkldghdfghdfghdfhdfnjkh"));
```
Comment (Bloodhount) [inline ConsoleApp_06_avgust_2021_practic/Program.cs:30]: this is also a good option

**Review by Bloodhount — CHANGES_REQUESTED** [review]
end review

**Comment by tester101010** [comment]
ok merge

**Pull request merged**